index.html

```diff
@@ -1,75 +1,88 @@
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+                <img src="img/jumbo.jpg" id="jumbo-img" />
+            </header>
 
-        <section class="bottom-content">
+            <section class="top-content">
+                <div class="text-container">
+                    <h2>The Future</h2>
+                    <p>
+                        Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam
+                        condimentum venenatis magna eget finibus.
+                    </p>
+                </div>
+                <div class="text-container">
+                    <h2>The Past</h2>
+                    <p>
+                        Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam
+                        condimentum venenatis magna eget finibus.
+                    </p>
+                </div>
+            </section>
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
+            <section class="middle-content">
+                <h2>Why Did It Have To Be Boxes...</h2>
+            <!-- I couldn't think of an elegant solution, so I just did in-line style changes -->
+                <div class="boxes">
```

Comment:
While inline styles work, the concept here is using parent/child relationships to finish building out the boxes.  There are multiple ways to do this.  You can use multiple classes, ids or CSS selectors to get the job done.